examples/500-cli/cli.md

```diff
@@ -364,21 +364,21 @@ there's only one record in this map.
 
 ## Catalog Operations
 
-[testmark]:# (catalog/fs/.warpforge/root)
+[testmark]:# (catalog/tags=net/fs/.warpforge/root)
 ```
 this file marks the workspace as a root workspace
 ```
 
 ### Initialize a Catalog
 
-[testmark]:# (catalog/sequence)
+[testmark]:# (catalog/tags=net/sequence)
 ```
 warpforge catalog init my-catalog
 ```
 
 ### List Catalogs
 
-[testmark]:# (catalog/then-ls/sequence)
+[testmark]:# (catalog/tags=net/then-ls/sequence)
```

Comment:
I like having this "tags=" thing here for legibility.  We should've done this the first time around, and I'm happy to see it now.

Reply:
Yeah. It occurs to me that for testmark parsing purposes the tags must _always_ be in the same order. Might annoy some people but it's probably fine. Makes it easy to sed replace later at least.

Reply:
+1 to "We should've done this the first time around" :+1: